.github/workflows/main.yaml

```diff
@@ -15,15 +15,25 @@ jobs:
       TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   helm-release:
-    runs-on: ubuntu-latest
-    steps:
-      - run: echo "placeholder"
+    uses: naviteq/github-actions/.github/workflows/create-helm-release.yaml@main
+    with:
+      CHART_LOCATION: helm
+      UPDATE_BRANCH: main
 
   helm:
+    name: Build Helm chart
     runs-on: ubuntu-latest
-    steps:
-      - run: echo "placeholder"
     needs: [helm-release]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+      - name: Build
+        run: |
```

Comment:
Let's include that in our action instead

Reply:
I assume we'd need to create a separate action in https://github.com/naviteq/github-actions ?

Reply:
Created it at
https://github.com/naviteq/github-actions/pull/16
Tests pass: 
https://github.com/naviteq/curator-opensearch/actions/runs/3685945374